code/modules/reagents/reactions/reaction_herbal.dm

```diff
@@ -1,7 +1,7 @@
 /decl/chemical_reaction/drug/herbal
 	abstract_type       = /decl/chemical_reaction/drug/herbal
 	result_amount       = 2
-	minimum_temperature = 100 CELSIUS
+	minimum_temperature = 50 CELSIUS
```

Comment:
i don't thiiink there was a specific reason 50C was chosen aside from the fact that it's just halfway to 100C. i feel like maybe the tea ones should require 100C (or close to it) still, and the tincture ones should have a lower temperature. i mean i guess there's no reason it can't just be arbitrary, this isn't really a big deal, just thoughts